packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -131,12 +131,15 @@ class CupertinoSwitch extends StatefulWidget {
 class _CupertinoSwitchState extends State<CupertinoSwitch> with TickerProviderStateMixin {
   @override
   Widget build(BuildContext context) {
-    return _CupertinoSwitchRenderObjectWidget(
-      value: widget.value,
-      activeColor: widget.activeColor ?? CupertinoColors.activeGreen,
-      onChanged: widget.onChanged,
-      vsync: this,
-      dragStartBehavior: widget.dragStartBehavior,
+    return Opacity(
+      opacity: widget.onChanged == null ? 0.4 : 1.0,
```

Comment:
Document how you got to this value (it could just be eyeballing). This helps our next maintainer figure out how confident he/she needs to be before changing this.